documentation/source/development/platforms.rst

```diff
@@ -0,0 +1,31 @@
+Supported platforms
+===================
+
+- Vulkan API is completely platform-agnostic, which allows it to run on various operating systems.
+- The required drivers for Vulkan are usually part of your graphic card's drivers.
+- Update your graphics drivers ase often as possible since new drivers with Vulkan updates are released frequently.
+- Driver updates contain new features, bug fixes, and performance improvements.
+- Check out `Khronos website <https://www.khronos.org/vulkan/>`__ for more information.
+
+Microsoft Windows
+-----------------
+
+- We support x64 Microsoft Windows 7, 8, 8.1 and 10.
+
+Linux
+------
+
+- We support every x64 Linux distribution for which Vulkan drivers exist.
```

Comment:
And it must support the Vulkan extensions which are required by the engine? Or does every Vulkan supported GPU have them. (e.g. we might introduce raytracing)

Reply:
Vulkan extensions are system specific and GPU specific. Ideally, Inexor-engine should work without any extensions at all since they should be optional features.